cmd/root.go

```diff
@@ -30,7 +30,7 @@ import (
 
 var (
 	rootCmd = &cobra.Command{
-		Use:   dist.Exec,
+		Use:   dist.AnalyzeExec,
```

Comment:
Where is the `WSWebsite` constant leveraged from?

Reply:
Honestly, I wasn't able to find it.  It even compile without it.

Reply:
I bet it was added in an intermediate commit, or as a 'we will use this in the future'. Either case it should be fine to leave